### ISSUE TITLE
Drupal CMS MVP of Vale as plain language linter 

### ISSUE THREAD

**Issue by meganhkelley** [department-of-veterans-affairs/va.gov-team#2233]
### User Story

As a VFS team member writing content for VA.gov, I want to see automated content validations, so that I am better able to self-serve and write error-free, plain language content.

### Assumptions and Hypotheses

We **assume** that VFS teams will engage with automated content validations, and make changes based on the warnings shown.
We **hypothesize** that if we emphasize and build for self-service, then the platform will be more scalable and able to support more teams building on VA.gov.
We **hypothesize** that if we emphasize and build for self-service of content writing, via improved documentation and automated tests, then the burden of manual reviews on VSP's content editor will decrease and the back-and-forth of content reviews will decrease (better experience for VSP and VFS).

### Background

This epic tracks work required to implement a basic MVP of the plain language linting tool. 

The MVP approach is meant to:

- lint content from the VA.gov CMS, using a very basic ruleset, as a proof of concept
- verify our assumptions and identify how the user experience will be affected
- uncover obstacles or hidden pain points before investing more effort

**Comment by meganhkelley** [comment]
Meeting notes 11/26 (Megan, Keifer, Alex, Bill): 

Updates from Keifer: 
- Tool is not yet complete but making progress! Worked through some obstacles with it past couple days.
- It will “live” on the preview page for the CMS editor, same places as the accessibility tool that Trevor made (will be encountered by the same users in the same place)
- There is no pathway to implementing content changes for `vets-website`, but if we wanted to put this there it would probably be lower lift than this. There’s a GitHub action that could make this very easy.

Measuring success: 
- Megan’s ideal — to measure whether anyone makes any changes to their content based on validations we serve (measure the errors they see on the same content over time?) 
- Keifer and Bill to consider how we might get quantitative data on above — errors per node over time? 
- *Decision of initial success metric:* for MVP, we just want to validate that we have put the tool in a place that makes sense for the users we want to hit. Therefore we will create and distribute a qualitative survey a few days after launch to assess whether users are seeing the tool, whether it makes sense in their workflow, and whether they self-report having made any content changes based on its validations.

**Comment by kfrz** [comment]
> There is no pathway to implementing content changes for vets-website, but if we wanted to put this there it would probably be lower lift than this. There’s a GitHub action that could make this very easy.

There are options we should likely consider for implementing content linting on `vets-website` pull requests. This is not something we're working on for this epic, we're focused on the content from Drupal.  

@billfienberg is [this](https://github.com/marketplace/actions/vale-lint) the action you were mentioning re: test repo? If so we could ticket up that work, one way or another. We could at least write an RFC with regards to that change.

**Comment by meganhkelley** [comment]
1/7/20 status update and decision: 

Status: Code is live in vets-website for these validations to show adjacent to the CMS user's workflow (as part of preview using node ID). Need to coordinate with CMS team to deploy this (so that CMS users can actually see the validations when previewing using a node ID). 

Decision: The MVP of plain language linting is pivoting to use a GitHub action as a proof of concept, instead of the initial plan to launch this feature adjacent to the Drupal CMS user's workflow (in preview using the node ID). CMS integrations will be put on hold until we have shipped and gathered feedback on this tool as a GitHub action.

Reason: Per VSP content editor Peggy's recommendation. Most CMS users at this point are content writers and editors, they do not need excessive support from us. The teams who need more support are those who are not specializing in content, who are inputting their content directly into their VA.gov products. Also, although we have completed a fair bit of work on the initial CMS MVP, there would still be a fair bit of troubleshooting left/coordination with the CMS team to get that working.

Documented: https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/content/automated-content-review/decisions.md
